Umbraco-Cloud/Getting-Started/Migrate-Existing-Site/index.md

```diff
@@ -153,44 +110,33 @@ If you on your existing site have been working with members and made changes to
 
 :::note
 
-### Data types
-
-Have you been using _older_ data types on your project, you will need to go through a few steps in order to avoid running into collision errors when deploying your migrated sites to Umbraco Cloud.
-
-The data types in question are: Content Picker, Media Picker, Member picker, Multiple Media Picker (using Media Picker) and Related Links.
-
-You can either
-
-1. Rename the old data types on your existing site or
-2. Follow the steps in the [Colliding Data types](../../Troubleshooting/Deployments/Colliding-Datatypes) article
-:::
-
 ## 5. Clone down the Cloud project
 
 With your Umbraco Cloud project ready for the migration, it is time to clone down the project to your local machine.
 
-<iframe width="800" height="450" src="https://www.youtube.com/embed/1uUadErk9vQ?rel=0" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
-
 Run the site locally and verify your own project and the cloned Umbraco Cloud project are using the same Umbraco version. After you've verified this, shut down the site, and it's now time to start merging the two projects.
 
 ## 6. Move and merge files
 
 Merging your existing site into the Umbraco Cloud project is a matter of moving and merging files between the two projects. When following the steps outlined below it is important that you do not overwrite any settings and configuration related to Umbraco Deploy.
```

Comment:
Maybe add this in an important tag as a note, tip

When following the steps outlined below it is important that you do not overwrite any settings and **configurations** related to Umbraco Deploy.